Forwarded: not-needed
Author: Otto Kekäläinen <otto@debian.org>
Subject: Stop using versioned package names also in CMake when it runs CPack
  Make CPack install files in `/usr/share/doc/mariadb-server` instead of old
  `/usr/share/doc/mariadb-server-10.6`.
  .
  This is already done in upstream (during 10.9 development cycle) so there
  is no need to submit this change upstream.

--- a/cmake/install_macros.cmake
+++ b/cmake/install_macros.cmake
@@ -140,7 +140,7 @@ FUNCTION(INSTALL_DOCUMENTATION)
   IF(RPM)
     SET(destination "${destination}/MariaDB-${group}-${VERSION}")
   ELSEIF(DEB)
-    SET(destination "${destination}/mariadb-${group}-${MAJOR_VERSION}.${MINOR_VERSION}")
+    SET(destination "${destination}/mariadb-${group}")
   ENDIF()
 
   INSTALL(FILES ${files} DESTINATION ${destination} COMPONENT ${ARG_COMPONENT})
